user_modules/IsReady_Template/testCasesForNavi_IsReady.lua

```diff
@@ -0,0 +1,273 @@
+local testCasesForNavi_IsReady = {}
+
+require('atf.util')
+local events = require("events")
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+
+--[[@InitHMI_onReady_without_Navi_IsReady_available_false: replace original InitHMIOnReady from connecttest
+--! without expect Navi.IsReady
+--! @parameters: exp_occur - times to wait Navi.<<RPC>>
+--! in case Navi.IsReady(available = false), exp_occur = 0, else exp_occur = 1
+--]]
+function testCasesForNavi_IsReady.InitHMI_onReady_without_Navi_IsReady(self, exp_occur)
+  if(exp_occur == nil) then exp_occur = 0 end
+
+  local function ExpectRequest(name, mandatory, params)
+    local event = events.Event()
+    event.level = 2
+    event.matches = function(_, data) return data.method == name end
+    return
+      EXPECT_HMIEVENT(event, name)
+      :Times(mandatory and 1 or AnyNumber())
+      :Do(function(_, data)
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", params)
+      end)
+  end
+
+  ExpectRequest("BasicCommunication.MixingAudioSupported",
+    true,
+    { attenuatedSupported = true })
+  ExpectRequest("BasicCommunication.GetSystemInfo", false,
+    {
+      ccpu_version = "ccpu_version",
+      language = "EN-US",
+      wersCountryCode = "wersCountryCode"
+    })
+  ExpectRequest("UI.GetLanguage", true, { language = "EN-US" })
+  :Times(exp_occur)
+  :Timeout(20000)
+
+  ExpectRequest("VR.GetLanguage", true, { language = "EN-US" })
+  ExpectRequest("TTS.GetLanguage", true, { language = "EN-US" })
+  ExpectRequest("UI.ChangeRegistration", false, { }):Pin()
+  ExpectRequest("TTS.SetGlobalProperties", false, { }):Pin()
+  ExpectRequest("BasicCommunication.UpdateDeviceList", false, { }):Pin()
+  ExpectRequest("VR.ChangeRegistration", false, { }):Pin()
+  ExpectRequest("TTS.ChangeRegistration", false, { }):Pin()
+  ExpectRequest("VR.GetSupportedLanguages", true, {
+      languages = {
+        "EN-US","ES-MX","FR-CA","DE-DE","ES-ES","EN-GB","RU-RU",
+        "TR-TR","PL-PL","FR-FR","IT-IT","SV-SE","PT-PT","NL-NL",
+        "ZH-TW","JA-JP","AR-SA","KO-KR","PT-BR","CS-CZ","DA-DK",
+        "NO-NO","NL-BE","EL-GR","HU-HU","FI-FI","SK-SK" }
+    })
+  ExpectRequest("TTS.GetSupportedLanguages", true, {
+      languages = {
+        "EN-US","ES-MX","FR-CA","DE-DE","ES-ES","EN-GB","RU-RU",
+        "TR-TR","PL-PL","FR-FR","IT-IT","SV-SE","PT-PT","NL-NL",
+        "ZH-TW","JA-JP","AR-SA","KO-KR","PT-BR","CS-CZ","DA-DK",
+        "NO-NO","NL-BE","EL-GR","HU-HU","FI-FI","SK-SK" }
+    })
+  ExpectRequest("UI.GetSupportedLanguages", true, {
+      languages = {
+        "EN-US","ES-MX","FR-CA","DE-DE","ES-ES","EN-GB","RU-RU",
+        "TR-TR","PL-PL","FR-FR","IT-IT","SV-SE","PT-PT","NL-NL",
+        "ZH-TW","JA-JP","AR-SA","KO-KR","PT-BR","CS-CZ","DA-DK",
+        "NO-NO","NL-BE","EL-GR","HU-HU","FI-FI","SK-SK" }
+    })
+  :Times(exp_occur)
```

Comment:
@vvvakulenko, as UI.IsReady will come, expectation for UI.GetLanguage should be removed.
You should have only "ExpectRequest("UI.GetSupportedLanguages", true, {.....})", so please remove lines 68-69